c7n/resources/sqs.py

```diff
@@ -193,6 +193,121 @@ def process_resource(self, client, resource):
                 'Statements': found}
 
 
+@SQS.action_registry.register('add-statements')
+class AddPolicyStatements(AddPolicyBase):
```

Comment:
this can be removed, the use case for it is handled by modify

Reply:
I removed this, should I also remove the AddPolicyBase class in policy.py or would you like me to keep that in the PR for future possible use?

Reply:
Removed